website/docs/language/state/purpose.mdx

```diff
@@ -35,8 +35,8 @@ support tags.
 Therefore, for mapping configuration to resources in the real world,
 Terraform uses its own state structure.
 
-Terraform expects that each remote object is bound to only one resource
-instance, which is normally guaranteed by Terraform being responsible for
+Terraform expects that each remote object on the provider is bound to only one resource
```

Comment:
"on the provider" is ambiguous here. We could change this to "on the service provider," but I think it can be removed entirely -- "remote object" is clear enough.